packages/contractkit/src/wrappers/SortedOracles.ts

```diff
@@ -124,9 +124,12 @@ export class SortedOraclesWrapper extends BaseWrapper<SortedOracles> {
    */
   async removeExpiredReports(
     token: CeloToken,
-    numReports: number
+    numReports?: number
   ): Promise<CeloTransactionObject<void>> {
     const tokenAddress = await this.kit.registry.addressFor(token)
+    if (!numReports) {
+      numReports = (await this.getReports(token)).length - 1
+    }
```

Comment:
Nice! This will simplify the logic in the oracle client as well. Other than the case when _all_ reports are expired, it's hard to imagine a scenario where we don't want to remove all the expired ones. So, this seems like the best default behavior.